control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -967,3 +966,23 @@ func buildKonnectivityVolumeClusterCerts(v *corev1.Volume) {
 		DefaultMode: pointer.Int32(0640),
 	}
 }
+
+func renderAuditLogScript(auditLogFilePath string) string {
```

Comment:
Could we please export the function and re-use it in the OAPI deployment?

Reply:
Done!